include/extractor/turn_instructions.hpp

```diff
@@ -31,37 +36,80 @@ enum class TurnInstruction : unsigned char
     AccessRestrictionPenalty = 129
 };
 
+namespace
+{
+ //shiftable turns to left and right
+const bool shiftable_left[] = {false, false, true, true, true, false, false, true, true};
+const bool shiftable_right[] = {false, false, true, true, false, false, true, true, true};
+} // namespace
```

Comment:
anon. ns in header is not a good idea since every TU gets a copy of the vars: just make them const constexpr